src/events/request/RequestUnresolveEventHandler.ts

```diff
@@ -11,12 +11,22 @@ export default class RequestUnresolveEventHandler implements EventHandler<'messa
 
 	private logger = log4js.getLogger( 'RequestUnresolveEventHandler' );
 
+	private readonly botUserId: string;
+
+	constructor( botUserId: string ) {
+		this.botUserId = botUserId;
+	}
+
 	// This syntax is used to ensure that `this` refers to the `RequestUnresolveEventHandler` object
 	public onEvent = async ( { emoji, message }: MessageReaction, user: User ): Promise<void> => {
 		this.logger.info( `User ${ user.tag } removed '${ emoji.name }' reaction from request message '${ message.id }'` );
```

Comment:
Here also, maybe differentiate between request and non-request messages in the log.